src/pages/Borrow.js

```diff
@@ -111,4 +111,4 @@ function Auth() {
   );
 }
 
-export default hot(Auth);
+export default hot(Borrow);
```

Comment:
Renaming this file made sense to me, `Auth` doesn't describe it quite so well now that we have included Savings